### DISCUSSION TITLE
Thread count keeps increasing when using ManagedExecutor for async calls in Quarkus

### DISCUSSION THREAD

**Discussion by BananKoenngen** [quarkusio/quarkus#43874] [Q&A]
DELETED

**Comment by cescoffier** [comment]
Any idea from where comes these managed executors?

**Reply by BananKoenngen** [reply]
> Any idea from where comes these managed executors?

What do you mean?

**Reply by cescoffier** [reply]
The Managed Executors, how are they configured?
